lib/ansible/modules/cloud/amazon/ec2_vpc_nacl_facts.py

```diff
@@ -60,7 +60,7 @@
 '''
 
 RETURN = '''
-nacl:
+nacls:
```

Comment:
Note: this isn't an interface change, but the docs incorrectly had this listed as `nacl` when it's always been `nacls`.

Reply:
No action needed here - just wanted that note on the PR record.